assets/js/base/utils/render-frontend.js

```diff
@@ -4,22 +4,41 @@
 import { render, Suspense } from '@wordpress/element';
 import BlockErrorBoundary from '@woocommerce/base-components/block-error-boundary';
 
+// Some blocks take care of rendering their inner blocks automatically. For example,
+// the empty cart. In those cases, we don't want to trigger the render function of
+// inner components on load. Instead, the wrapper block can trigger the event
+// `wc-blocks_render_blocks_frontend` to render its inner blocks.
+const selectorsToSkipOnLoad = [ '.wp-block-woocommerce-cart' ];
```

Comment:
For now, I'm hard-coding the Cart class name here, but if we see this is needed in other blocks, we could try to dynamically fill this array when blocks are rendered with the Data Registry.